monitor/index.ts

```diff
@@ -21,9 +21,11 @@ interface CheckSealFieldState {
   sleepStreakAlertLevel: number;
 }
 
-async function sendNotice(error: CodeChainAlert, targetEmail: string) {
+function sendNotice(error: CodeChainAlert, targetEmail: string) {
```

Comment:
I want to know the reason behind this change

Reply:
`async function` in javascript is a function that returns `Promise`, and it's strongly recommended to consume the rejected promise.
So calling `sendNotice` should be `sendNotice(...).catch(...)` but it's not convenient to use because the caller should take care of it.
The other way is this fix. Consuming the rejected promise in the function and don't expose the promise.

Reply:
Thank you for the explanation.